core/sdk/src/api/config.rs

```diff
@@ -124,3 +124,5 @@ impl ExtnPayloadProvider for LauncherConfig {
         RippleContract::Config
     }
 }
+
+pub fn something() {}
```

Comment:
Do we need this method

Reply:
No we do not. It was a test for the pre-commit hook. 🤦 

Sorry, I didn't self-review this properly.